terraform/context.go

```diff
@@ -76,6 +77,23 @@ type Context struct {
 	runCh               <-chan struct{}
 }
 
+func parseVariableAsHCL(input string) (interface{}, error) {
+	const sentinalValue = "SENTINAL_TERRAFORM_VAR_OVERRIDE_KEY"
```

Comment:
Sorry for this total nit-pick, but the correct spelling of this word is "sentinel".